core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -82,12 +82,20 @@ public Authorizations convert(String value) {
   }
 
   public static class Password {
-    public byte[] value;
+    private byte[] value;
```

Comment:
This should likely be private final, otherwise, you'll need to deal with the correctness of copying a reference to an array.